js/phenogrid.js

```diff
@@ -1562,6 +1578,27 @@ var TooltipRender = require('./tooltiprender.js');
 			});
 			return res;
 		},
+		
+        // generic ajax POST
+        _ajaxPostData: function (target, url, postData) {
+            var self = this;
+            var res;
+            jQuery.ajax({
```

Comment:
We need to use $ instead of jQuery in phenogrid.js. After using require('jquery') it's exported to the $, so you will get jQuery is not defined in the console.